tide_search.module

```diff
@@ -13,19 +13,23 @@ use Drupal\search_api\IndexInterface;
 function tide_search_search_api_index_items_alter(IndexInterface $index, array &$items) {
   // Get any fields of type date and format it's value to align with RFC-3339.
   $index_fields = $index->getFields();
-  $date_fields = [];
-  foreach ($index_fields as $index_field) {
-    if ($type = $index_field->getType() === 'date') {
-      $date_fields[] = $index_field;
+  $date_field_ids = [];
+  foreach ($index_fields as $field_id => $index_field) {
+    if ($index_field->getType() === 'date') {
+      $date_field_ids[$field_id] = $field_id;
     }
   }
-  foreach ($items as $item_id => $item) {
-    foreach ($date_fields as $date_field) {
-      $date = $item->getField($date_field->getFieldIdentifier());
-      if (isset($date) && !empty($date->getValues())) {
-        $value = $date->getValues();
-        $date->setValues([_tide_search_get_formatted_date($value[0])]);
```

Comment:
This line was problematic. It replaces *ALL values* of the date field with the first value.